.github/workflows/benchmarks.yml

```
@@ -34,7 +34,7 @@ jobs:
         rust-version: ['1.42.0']  # Note: unused since controlled by `rust/rust-toolchain`
         js-version: ['16']
         r-version: ['4.1.2']
-        lua-version: ['2.0.5']  # Note: Not used as benchmark is broken.
+        lua-version: ['2.1.0']  # Note: unused since lua distribution manually downloaded
```

```suggestion
        lua-version: ['latest']  # Note: unused since lua distribution manually downloaded
```

To match `lua-install.sh`, so this will be true in the future as well.

---

Sure!